crates/sui-types/src/messages.rs

```diff
@@ -2220,11 +2223,11 @@ pub struct QuorumDriverRequest {
     pub request_type: QuorumDriverRequestType,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug)]
+#[derive(Clone, Debug)]
 pub enum QuorumDriverResponse {
     ImmediateReturn,
-    TxCert(Box<CertifiedTransaction>),
-    EffectsCert(Box<(CertifiedTransaction, CertifiedTransactionEffects)>),
+    TxCert(Box<VerifiedCertificate>),
```

Comment:
I don't think you should send VerifiedCertificate over the wire. The receiver end over the wire has no way to know if it's really verified or not.

Reply:
this struct is not sent over wire (that's why it no longer derive `Serialize` and `Deserialize`. [This thing](https://github.com/MystenLabs/sui/blob/make-verified-envelope-from-auth-agg-2/crates/sui-types/src/messages.rs#L2201) is.

Reply:
We have that semantic there, a `VerifiedEnvelop` is not serde-able.